aws-android-sdk-s3-test/src/androidTest/java/com/amazonaws/auth/policy/S3PolicyIntegrationTest.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2010-2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ *    http://aws.amazon.com/apache2.0
+ *
+ * This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.amazonaws.auth.policy;
+
+import com.amazonaws.auth.policy.actions.S3Actions;
+import com.amazonaws.auth.policy.conditions.ConditionFactory;
+import com.amazonaws.auth.policy.conditions.S3ConditionFactory;
+import com.amazonaws.auth.policy.conditions.StringCondition;
+import com.amazonaws.auth.policy.conditions.StringCondition.StringComparisonType;
+import com.amazonaws.auth.policy.resources.S3BucketResource;
+import com.amazonaws.services.s3.S3IntegrationTestBase;
+import com.amazonaws.services.s3.model.CannedAccessControlList;
+
+import org.junit.After;
+import org.junit.Test;
+
+/**
+ * Integration tests for the service specific access control policy code
+ * provided by the S3 client.
+ */
+public class S3PolicyIntegrationTest extends S3IntegrationTestBase {
```

Comment:
Since these tests don't require context, keep them under test/ folder. Try creating separate packages for integration and unit tests.